CHANGES.rst

```diff
@@ -1,3 +1,9 @@
+0.8.0 (unreleased)
+==================
+
+- Added ``pytest-mpl``, ``pytest-filter-subpackage`` and ``pytest-cov``
+  as dependencies. [#29]
```

Comment:
Changelog should be expanded with things like hypothesis and Python 2.7 and 3.5 support.

Reply:
Done!